patientsearch/src/js/components/PatientListTable.js

```diff
@@ -978,6 +978,7 @@ export default function PatientListTable() {
             handleLaunchApp(rowData);
           }}
           editable={{
+            isDeleteHidden: () => !appSettings["ENABLE_DELETE"],
```

Comment:
@achen2401  I don't see other references to `isDeleteHidden` in the code...

Reply:
sorry, @mcjustin  it is not clear from the code, but `isDeleteHidden` is one of the configurable option for the table component to remove the trashcan icon.  Here is the [reference](https://material-table.com/#/docs/features/editable)

Reply:
here is a screenshot of when I set the config `ENABLE_DELETE` to false:
![Screen Shot 2022-12-01 at 10 03 11 AM](https://user-images.githubusercontent.com/12942714/205127311-a39b5d6d-89e1-4c04-8f27-4bc786a51962.png)